src/rewrite.jl

```diff
@@ -372,11 +372,7 @@ end
 
 function _start_summing(::Nothing, first_term::Function)
     variable = gensym()
-    return Expr(
-        :block,
-        :($variable = MutableArithmetics.Zero()),
-        first_term(variable),
-    )
+    return Expr(:block, Expr(:(=), variable, Zero()), first_term(variable))
```

Comment:
Why don't we need `MA.Zero` ? `import` is like `using` for exported symbols ? Then it's not in line with the style guide rule of not doing `using` except for throw-away scripts.

Reply:
`:($variable = MutableArithmetics.Zero())` doesn't interpolate the _value_ `MutableArithmetics.Zero`, instead it is a symbol that needs to get looked up in the local scope of the rewritten expression.

In contrast, the `Expr(:(=), variable, Zero())` embeds the value `Zero()` into the expression that doesn't require a lookup, and it doesn't need a `MA.` prefix because `Zero()` is evaluated in the current module (`MutableArithmetics`) scope.